widgetssdk/src/main/java/com/glia/widgets/engagement/EngagementRepository.kt

```diff
@@ -47,7 +47,10 @@ internal interface EngagementRepository {
     fun initialize()
     fun reset()
     fun resetQueueing()
-    fun endEngagement(endedBy: EndedBy)
+    //used for ending engagement after visitor ends engagement explicitly
+    fun endEngagement()
+    //used for ending engagement after integrator ends the engagement from outside of SDK, for example [GliaWidgets.endEngagement]
```

Comment:
The comment is a bit confusing to me because it says `from outside of SDK` but mentions `[GliaWidgets.endEngagement]` which is an SDK function. Maybe it makes sense to rephrase it.